cloud_sql/mysql_instance_custom_subject_alternative_names/main.tf

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+# [START cloud_sql_instance_service_identity]
+resource "google_project_service_identity" "default" {
+  provider = google-beta
+  service  = "sqladmin.googleapis.com"
+}
+# [END cloud_sql_instance_service_identity]
+
+# [START privateca_ca_pool_suffix]
+resource "random_string" "default" {
+  length  = 10
+  special = false
+  upper   = false
+}
+# [END privateca_ca_pool_suffix]
+
+# [START cloud_sql_mysql_instance_ca_pool]
+resource "google_privateca_ca_pool" "default" {
+  name     = "customer-ca-pool-${random_string.default.result}"
+  location = "asia-northeast1"
+  tier     = "DEVOPS"
+  publishing_options {
+    publish_ca_cert = false
+    publish_crl     = false
+  }
+}
+# [END cloud_sql_mysql_instance_ca_pool]
+
+# [START cloud_sql_mysql_instance_ca]
+resource "google_privateca_certificate_authority" "default" {
```

Comment:
(Optional) Try to add description for each resource that has 4+ lines.

A short description of `why` help readers to understand faster!